src/helics/core/ActionMessageDefintions.hpp

```diff
@@ -331,7 +333,9 @@ across different compilers*/
 #define CMD_SET_GLOBAL action_message_def::action_t::cmd_set_global
 
 #define CMD_SEND_COMMAND action_message_def::action_t::cmd_send_command
+#define CMD_SEND_COMMAND_ORDERED action_message_def::action_t::cmd_send_command_ordered
 #define CMD_COMMAND_STATUS action_message_def::action_t::cmd_command_status
+#define CMD_COMMAND_STATUS_ORDERED action_message_def::action_t::cmd_command_status
```

Comment:
Where is `cmd_command_status` defined? I'm not seeing it in the action_t enum in this file.